docs/admin/authentication.md

```diff
@@ -33,7 +33,9 @@ plane, must authenticate when making requests to the the API server.
 ## Authentication strategies
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
-authenticate API requests through authentication plugins. As HTTP request are
+authenticate API requests through authentication plugins. A bearer token
+must be something that can be included in an HTTP header value without any
```

Comment:
Another way to solve the organizational problem is to put the full syntactic considerations in the earliest section about bearer tokens (which is currently "Static Token File"), and the later bearer token sections can refer back to that.